docs/shared/recipes/nx-release/getting-started-with-nx-release.md

```diff
@@ -0,0 +1,273 @@
+❯ major
+  premajor
+  minor
+  preminor
+  patch
+  prepatch
+  prerelease
+  Custom exact version
+```
+
+### Previewing the Results
+
+After this prompt, the command will finish, showing you the preview of changes that would have been made if the `--dry-run` option was not passed.
+
+```{% command="nx release --first-release --dry-run" %}
+
+ >  NX   Running release version for project: pkg-1
+
+pkg-1 🔍 Reading data for package "@myorg/pkg-1" from packages/pkg-1/package.json
+pkg-1 📄 Resolved the current version as 0.0.1 from packages/pkg-1/package.json
+✔ What kind of change is this for the 3 matched projects(s)? · patch
+pkg-1 ✍️  New version 0.0.2 written to packages/pkg-1/package.json
+
+ >  NX   Running release version for project: pkg-2
+
+pkg-2 🔍 Reading data for package "@myorg/pkg-2" from packages/pkg-2/package.json
+pkg-2 📄 Resolved the current version as 0.0.1 from packages/pkg-2/package.json
+pkg-2 ✍️  New version 0.0.2 written to packages/pkg-2/package.json
+pkg-2 ✍️  Applying new version 0.0.2 to 1 package which depends on pkg-2
+
+ >  NX   Running release version for project: pkg-3
+
+pkg-3 🔍 Reading data for package "@myorg/pkg-3" from packages/pkg-3/package.json
+pkg-3 📄 Resolved the current version as 0.0.1 from packages/pkg-3/package.json
+pkg-3 ✍️  New version 0.0.2 written to packages/pkg-3/package.json
+
+UPDATE packages/pkg-1/package.json [dry-run]
+
+    "name": "@myorg/pkg-1",
+-   "version": "0.0.1",
++   "version": "0.0.2",
+    "dependencies": {
+      "tslib": "^2.3.0",
+-     "@myorg/pkg-2": "0.0.1"
++     "@myorg/pkg-2": "0.0.2"
+    },
+
+UPDATE packages/pkg-2/package.json [dry-run]
+
+    "name": "@myorg/pkg-2",
+-   "version": "0.0.1",
++   "version": "0.0.2",
+    "dependencies": {
+
+UPDATE packages/pkg-3/package.json [dry-run]
+
+    "name": "@myorg/pkg-3",
+-   "version": "0.0.1",
++   "version": "0.0.2",
+    "dependencies": {
+
+
+ >  NX   Updating npm lock file
+
+
+ >  NX   Staging changed files with git
+
+
+NOTE: The "dryRun" flag means no changes were made.
+
+ >  NX   Previewing an entry in CHANGELOG.md for v0.0.2
+
+
+CREATE CHANGELOG.md [dry-run]
++ ## 0.0.2 (2024-01-23)
++
++ This was a version bump only, there were no code changes.
+
+ >  NX   Staging changed files with git
+
+
+NOTE: The "dryRun" flag means no changelogs were actually created.
+
+ >  NX   Committing changes with git
+
+
+ >  NX   Tagging commit with git
+
+Skipped publishing packages.
+```
+
+### Run Without `--dry-run`
+
+If the preview looks good, run the command again without the `--dry-run` option to actually create the release.
+
+```shell
+nx release --first-release
+```
+
+The command will proceed as before, prompting for a version bump and showing a preview of the changes. However, this time, it will prompt you to publish the packages to the remote registry. If you say no, the publishing step will be skipped If you say yes, the command will publish the packages to the npm registry.
```

Comment:
```suggestion
The command will proceed as before, prompting for a version bump and showing a preview of the changes. However, this time, it will prompt you to publish the packages to the remote registry. If you say no, the publishing step will be skipped. If you say yes, the command will publish the packages to the npm registry.
```